hawc/apps/assessment/signals.py

```diff
@@ -45,3 +46,19 @@ def invalidate_endpoint_cache(sender, instance, **kwargs):
     SerializerHelper.clear_cache(
         apps.get_model("animal", "Endpoint"), {"assessment_id": instance.id}
     )
+
+
+@receiver(pre_save, sender=models.Job)
```

Comment:
do we need this method? I think you can add `default=uuid.uuid4` on the model class and that should do it

https://docs.djangoproject.com/en/3.1/ref/models/fields/#uuidfield